tests/test_control_panel.py

```diff
@@ -1,20 +1,36 @@
 # -*- coding: utf-8 -*-
 from unittest import mock
 
+import requests
+
 from tests import IntegrationTestCase
 from tests.fixtures import UserFactory
 
 
 @mock.patch('dashboard.service.fetch_user_data')
 class TestControlPanel(IntegrationTestCase):
```

Comment:
I still think that in test where we cannot see Control Panel, we should check if HTTP status code is 403